dcrtimed/config.go

```diff
@@ -477,6 +478,22 @@ func loadConfig() (*config, []string, error) {
 		cfg.WalletCert = path
 	}
 
+	if len(cfg.APIToken) > 0 {
+		var validTokens []string
+		for _, token := range cfg.APIToken {
+			token = strings.TrimSpace(token)
+
+			// Do not allow blank API tokens
+			if token == "" {
+				fmt.Printf("%v: Blank API token ignored\n", funcName)
```

Comment:
exit here.

Reply:
Wait, so each of the proxy instances must have the api keys on them as well, or can this be skipped for proxy mode? This now exits early, just double checking.

Reply:
dcrtimed should not start up if an invalid (empty, or any other future restrictions) apitoken is specified.

Reply:
The change has been made, and dcrtimed will not start w/ an invalid api key. 

In the context of this PR, are there additional requirements about how the proxy should handle its api key?

Current behavior is
* client sends an apikey
* proxy forwards apikey value to the backend (proxy apikey is ignored)
* backend checks the apikey